### DISCUSSION TITLE
[awc] How to get the URL after a redirection?

### DISCUSSION THREAD

**Discussion by rharish101** [actix/actix-web#3387] [Q&A]
If I request a webpage with a URL, and the website redirects to another page, how do I get the URL of the new page once awc returns from a `ClientRequest::send`? This is when redirections are allowed (i.e. not disabled when building the client) in the client.

**Comment by robjtede** [comment]
It's not currently possible, but it's a good idea!

I've created a feature request issue for this: #3403
